src/library_fs.js

```diff
@@ -1,6 +1,10 @@
 mergeInto(LibraryManager.library, {
   $FS__deps: ['$ERRNO_CODES', '$ERRNO_MESSAGES', '__setErrNo', '$PATH', '$TTY', '$MEMFS', '$IDBFS', '$NODEFS', 'stdin', 'stdout', 'stderr'],
   $FS__postset: 'FS.staticInit();' +
+                'Module["FS"] = Object.create(null);' +
```

Comment:
Let's put it on the FS object itself. We can set these values in `staticInit`, something like

````
  FS.filesystems = {
    'MEMFS' : MEMFS,
    ..
  };
````

Putting it on module is risky, as something else called "FS" might be there - the FS object itself, in fact, if the user asked to export it.

Side note, why not do `{}` instead of `Object.create(null)`? I'm not aware of benefits to the longer form.